src/utilities/manifest-utilities.ts

```diff
@@ -101,9 +101,12 @@ export function getManifestObjects(paths: string[]): any[] {
     let manifestContents = [];
     if (paths.length > 0) {
         paths.forEach((path) => {
-            let manifestContent = fileHelper.getManifestFileContents(path);
-            if (manifestContent) {
-                manifestContents.push(manifestContent);
+            let contents = fileHelper.getManifestFileContents(path);
+            if (contents &&
+                contents.length > 0) {
+                contents.forEach((manifestContent) => {

```

Comment:
Push the whole array? (iteration not required)
`manifestContents.push(contents);`